src/js/pages/Layout.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class Layout extends React.Component {
+  render() {
+    return (
+      <h1>KillerNews.net</h1>
+    );
+  }
+}
```

Comment:
you should not mix js and ts, because using js will destroy benefit of using ts.

and also you should not use class component because this is legacy and low performance

you should avoid from using default export